### PR TITLE
Updating openshift documentation

### PR DESCRIPTION
**Pull request by chrislovecnm**
Adding script to do iam commands and other commands.

### CODE REVIEW

**Review by keith-mcclellan — COMMENTED** [review]
With the latest changes I was able to get this working.  I'd like @johnrk to try this as well before we merge though.

File: docs/openshift/gcp-install.md [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+tar -xzf openshift-client-linux.tar.gz 
+chmod +x ./openshift-install
+chmod +x ./oc
+
+sudo mv ./openshift-install /usr/local/bin/openshift-install
+sudo mv ./oc /usr/local/bin/oc
+rm openshift-client-linux.tar.gz openshift-install-linux.tar.gz
+```
+
+## Configure GCP setting and Service Account
```
[the remark this answers is not in the document]
Comment (chrislovecnm) [reply docs/openshift/gcp-install.md:57]: > alternatively, I need to know the name of the project I should be using

That is internal information for CRL, so I would rather not include the GCP project id in the documentation, for security sake.

I can link the GCP docs on how to set the project, but admittedly the script sets the project it is using.  So technically if you have the correct project id, you should be able to use the script.

We are reading the project id via what is passed via the `-p` flag and set it here https://github.com/cockroachdb/cockroach-operator/pull/404/files#diff-23958cddb76eead2afdcf830620732e4117a43445dcbd1e82ed831130b826a1aR91 using `gcloud`.  We can mention using `gcloud projects list` and insert the project id.

[the remark this answers is not in the document]
Comment (chrislovecnm) [reply docs/openshift/gcp-install.md:57]: I add more documentation on how to object the full project id.

**Comment by chrislovecnm** [comment]
@keith-mcclellan can we please merge and then make changes? Small PR's 😁

**Review by keith-mcclellan — APPROVED** [review]
I was able to get this working.  LGTM

**Pull request merged**